modxcms/revolution
# Fix active state for files in main tree

### What does it do?
Made a couple tweaks, one of which ensures the active file value (pulled from the query string) in decoded for comparison to the node id.

### Why is it needed?
Active files were not being highlighted in the main tree.

### How to test
Clear your caches and remember to run `grunt build` within the __build/templates/default_ directory to compile the changes into the main js file. Then navigate to the _Files_ tab on the main tree and click to open files at various levels of the hierarchy. Now the file that is open should be highlighted in the tree. Note that the behavior remains correct for Resources and Elements.

### Related issue(s)/PR(s)
Resolves #15302.
